openflow-sdn/openflow-protocol.md

```diff
@@ -0,0 +1,65 @@
+# What is the OpenFlow protocol? 
+
+**Openflow protocol** is a standard in Software Defined Networking(SDN) architecture. This protocol defines the communication between an SDN controller and the network device/agent. Before delving deep into the OpenFlow protocol, here is a background on SDN. 
```

Comment:
suggestion
The **OpenFlow (OF) protocol** is a standard in Software Defined Networking (SDN) architecture. This protocol defines the communication between an SDN controller and the network device/agent. Before delving deep into the OpenFlow protocol, here is a background on SDN.